react/features/stream-effects/blur/index.js

```diff
@@ -21,5 +23,14 @@ export function createBlurEffect() {
         return Promise.reject(new Error('JitsiStreamBlurEffect not supported!'));
     }
 
+    // change the TF backend to use WebGL
+    TF.setBackend('webgl').then(() => {
+        logger.info('TensorFlow backend changed to use the WebGL');
+    })
+    .catch(error => {
+        logger.info('TensorFlow backend could not be changed to use WebGL, performance may suffer: ',
+            error);
+    });
```

Comment:
Do we really want to allow the non webgl blur to occur? Based on what I saw, I almost want to say we don't want it because it makes chrome unresponsive.

Reply:
Agreed, I'd say we reject the promise above and not continue.

Reply:
Ok, will reject the promise here then.